system/ui/lib/wifi_manager.py

```diff
@@ -398,15 +398,15 @@ def __init__(self):
 
     self._thread = threading.Thread(target=self._run, daemon=True)
     self._loop: asyncio.EventLoop | None = None
-    self._running = False
+    self._running = threading.Event()
```

Comment:
Let's rename it to `_stop_event`, too

Reply:
We usually have the logic for these the other way around, I think it's easier to quickly understand what's going on if we keep it consistent

We can change it to `_stop_event` instead and flip the logic around